legacy/README.md

```diff
@@ -0,0 +1,145 @@
+# Legacy docker-compose support
+
+This example demonstrates a setup that is compatible with the legacy version of Docker Compose.
```

Comment:
Yes, like in the headline of that section. Eventually it will be good to be ultra-specific and talk about:

> the old Docker Compose version 1.x, `docker-compose`